### ISSUE TITLE
Add Kernel Update Workaround to Bootstrap

### ISSUE THREAD

**Issue by briandoconnor** [ICGC-TCGA-PanCancer/cli#15]
sudo chmod a-x /etc/grub.d/30_os-prober

**Comment by briandoconnor** [comment]
Also this needs to be run on the worker nodes in Openstack too, need to add to Ansible script

**Comment by briandoconnor** [comment]
```
---
# file: install.yml
# This file is used for installing a container host

# Not sure why, but I needed this to create a /etc/hosts that actually works with the Ubuntu image here on openstack
- hosts: master:worker
  sudo: True
  # Idempotent way to build a /etc/hosts file with Ansible using your Ansible hosts inventory for a source.
  # Will include all hosts the playbook is run on.
  # Inspired from http://xmeblog.blogspot.com/2013/06/ansible-dynamicaly-update-etchosts.html
  tasks:
  - name: Kernel update issue workaround
    shell: sudo chmod a-x /etc/grub.d/30_os-prober
  - name: Build hosts file for all hosts
    lineinfile: dest=/etc/hosts regexp='.*{{ item }}$' line="{{ hostvars[item].ansible_default_ipv4.address }} {{item}}" state=present
    when: hostvars[item].ansible_default_ipv4.address is defined and docker is not defined
    with_items: groups['all']
  - name: Setting hostname to {{ inventory_hostname }}
    hostname: name={{ inventory_hostname }}
  - name: Disable automatic retrieval of hostname from dhcp on Amazon EC2
    template: src=roles/common/templates/dhclient.conf dest=/etc/dhcp/dhclient.conf
  - name: Disable setting hostname from /etc/hostname on Amazon EC2
    template: src=roles/common/templates/hostname.conf dest=/etc/init/hostname.conf
  - name: Disable ephemeral mount at /mnt for amazon AWS
    shell: umount /mnt
    register: umount_output
    ignore_errors: yes
  - name: Create directory for downloads
    file: path=/home/ubuntu/downloads owner=ubuntu group=ubuntu state=directory
```

**Comment by SolomonShorser-OICR** [comment]
@briandoconnor  I didn't stick around to watch the deployment. Could you please let me know if this worked and allowed installation of docker in OpenStack? I have an issue in container-host-bag (see: https://github.com/ICGC-TCGA-PanCancer/container-host-bag/issues/26) to track this, since that's where the fix will be added.

**Comment by SolomonShorser-OICR** [comment]
Fixed in container-host-bag release 1.0-rc.17
Commit: https://github.com/ICGC-TCGA-PanCancer/container-host-bag/commit/7329e62186f6cca0b84309d544a26be6c93dc290
